host/xtest/regression_4000.c

```diff
@@ -2409,6 +2427,241 @@ static void xtest_tee_test_4003_no_xts(ADBG_Case_t *c)
 	TEEC_CloseSession(&session);
 }
 
+#ifdef CFG_SECURE_KEY_SERVICES
+/* The PKCS#11 objects used for the tests */
+
+static CK_OBJECT_CLASS cktest_symkeyClass = CKO_SECRET_KEY;
+static CK_OBJECT_CLASS cktest_dataClass = CKO_DATA;
+
+static CK_KEY_TYPE cktest_aes_keyType = CKK_AES;
+static CK_KEY_TYPE cktest_gen_keyType = CKK_GENERIC_SECRET;
+
+static CK_BBOOL cktest_true = CK_TRUE;
+static CK_BBOOL cktest_false = CK_FALSE;
+
+#define	CK_CIPHERING_KEY_AES(_key_array) { \
+		{ CKA_ENCRYPT,	&cktest_true, sizeof(cktest_true) }, \
```

Comment:
You could consider using compound literals instead, like we doing here https://github.com/OP-TEE/optee_os/blob/master/ta/arch/arm/user_ta_header.c#L91-L105
The line would then become (with no extra dependencies):
```
{ CKA_ENCRYPT, &(CK_BBOOL){CK_TRUE}, sizeof(CK_BBOOL) }
```